README.md

```diff
@@ -2,6 +2,12 @@
 
 KimonoRSS is a tool for easily consuming JSON APIs from [KimonoLabs](https://www.kimonolabs.com) either in a ruby environment or as an RSS feed of entry history.
 
+## Developing
+
+You will need Postgres 9.3 or higher.  [Get Postgres.app](http://postgresapp.com/) and then use [the guide](http://postgresapp.com/documentation/cli-tools.html) to get set up.
+
```

Comment:
You know, I originally installed Postgres via homebrew, and I would agree, but the [Heroku setup](https://devcenter.heroku.com/articles/getting-started-with-rails4) guide [recommends the app](https://devcenter.heroku.com/articles/heroku-postgresql#local-setup).  

It turned out that doing it on homebrew I had the 9.4 client tools talking to a 9.1.3 server instance.  Which totally nuked my plan to do range types with the Schema model.  Took me a good hour to figure that out because `which psql` and `psql -V` was (obvs) telling me 9.4.

I (sadly) did `brew uninstall postgres` and followed this and it worked like a charm.  

TL;DR - n00b issues, following docs :/